pythonforandroid/bootstraps/sdl2/build/templates/AndroidManifest.tmpl.xml

```diff
@@ -63,6 +63,9 @@
                  android:theme="{{args.android_apptheme}}{% if not args.window %}.Fullscreen{% endif %}"
                  android:hardwareAccelerated="true"
                  android:extractNativeLibs="true" >
+                 
+        {{ args.extra_manifest_application_entry }}
```

Comment:
We should add this snippet to all the supported bootstraps.

Reply:
In **service_library**, **service_only** and **webview** not need,

Reply:
@dvjdjvu please do not resolve conversations, unless it's quite clear that are resolved.

Why this is not needed on `webview`, `service_library` and `service_only` ?

Reply:
I did not know what these things were for, now I have studied a little. I think you are right, it should be added.